### PR TITLE
Upgrade to richie 2.8.2 on all sites

### PR DESCRIPTION
**Pull request by sampaccoud**
### Changed
- Upgrade richie to 2.8.2

### Changed
- Upgrade richie to 2.8.2

### Changed
- Upgrade richie to 2.8.2

### Changed
- Upgrade richie to 2.8.2

### Changed
- Upgrade richie to 2.8.2
 
### Changed
- Upgrade richie to 2.8.2

### CODE REVIEW

File: sites/funcampus/src/frontend/scss/settings/_variables.scss [hunk 1]
```diff
@@ -158,6 +158,7 @@ $r-footer-logo-width-lg: 12.125rem;
 // full width like common blocks. Usefull if you plan to remove course run from
 // template
 $r-course-aside: null;
+$r-course-subheader-aside: null;
```
Comment (jbpenrath) [inline sites/funcampus/src/frontend/scss/settings/_variables.scss:161]: Nope this value should not be null.
```suggestion
$r-course-subheader-aside: 25%;
```

File: sites/funcorporate/src/frontend/scss/settings/_variables.scss [hunk 2]
```diff
@@ -158,6 +158,7 @@ $r-footer-logo-width-lg: 18.125rem;
 // full width like common blocks. Usefull if you plan to remove course run from
 // template
 $r-course-aside: null;
+$r-course-subheader-aside: null;
```
Comment (jbpenrath) [inline sites/funcorporate/src/frontend/scss/settings/_variables.scss:161]: Ditto
```suggestion
$r-course-subheader-aside: 25%;
```

File: sites/funmooc/src/frontend/scss/settings/_variables.scss [hunk 3]
```diff
@@ -158,6 +158,7 @@ $r-footer-logo-width-lg: 12.125rem;
 // full width like common blocks. Usefull if you plan to remove course run from
 // template
 $r-course-aside: 25%;
+$r-course-subheader-aside: 25%;
```
Comment (jbpenrath) [inline sites/funmooc/src/frontend/scss/settings/_variables.scss:161]: Didn't you want to increase the aside column width ? On the demo website, the value is `35%`.

Comment (sampaccoud) [reply sites/funmooc/src/frontend/scss/settings/_variables.scss:161]: hmm let's keep it for another PR when we will activate ecommerce...

File: sites/funcorporate/src/backend/templates/courses/cms/course_run_detail.html [hunk 4]
```diff
@@ -1,4 +0,0 @@
-{% extends "courses/cms/course_run_detail.html" %}
-
-{% comment %}Disable licences part. It does not make sens in FUN Corporate.{% endcomment %}
-{% block licenses %}{% endblock licenses %}
```
Comment (jbpenrath) [inline sites/funcorporate/src/backend/templates/courses/cms/course_run_detail.html:4]: Sure?

**Pull request merged**